api/provider_api.md

```diff
@@ -1,6 +1,6 @@
 <!--:nav_order:2-->
 
-# Adobe I/O Events Provider API
+# Adobe I/O Events Management API
 
 ## Prerequisites
```

Comment:
Can you use sub-bullets under 2 and 3, otherwise its a run-on sentence hard to understand, and looks ugly

Reply:
done https://github.com/AdobeDocs/adobeio-events/pull/77/commits/6fc89e86e53cc64a325bbbcbf278684d96cfc3d9